index.js

```diff
@@ -7,16 +7,12 @@ module.exports = (options) => {
   return {
     detect: (value, request) => {
       if (typeof value === 'object' && value !== null) {
```

Comment:
i think this line is not actually needed since drboom is checking if the value is an instanceof Error